### ISSUE TITLE
Megafauna - Fighting Firedrakes slowly accumulates lag and fps drop until the round ends

### ISSUE THREAD

**Issue by Taylorkills2** [tgstation/tgstation#49414]
## Reproduction: Be robust/immortal, fight for 4-6 minutes until you kill it. Notice minor increase in lag and minor drop in framerate. 
Go fight and kill another, notice heavy increase in lag and moderate drop in frames everywhere you go.
Fight and kill a third, lag is severe 
Try to fight and kill a fourth, your fps will probably be between 1-10 during the fight, I assume if you survive and win the lag and framerate drop will be even more severe, unable to testify due to my inability to kill a fourth.

Problem persists if you die and become a ghost.
Problem disappears after server restart.

**Comment by ATH1909** [comment]
Does this lag everyone, or just you?

**Comment by Taylorkills2** [comment]
> Does this lag everyone, or just you? Will spectators who are watching the fight also get increased amounts of lag? Will spectators who are just watching megafauna chilling out and not fighting anyone also get increased amounts of lag? Does this affect space dragon players (or sapient megafauna in general)?

Server wise? Just me.

I do not know, none have testified or told me, I have observed one player kill a fire drake myself,  but the lag and fps from the first drake is minor, noticeable but doesn't affect normal play.

I do not believe so, I have not noticed any lag when spectating and afking over megafauana.

I do not know about space dragons, but I have played sapient megafauna several times, and I did not notice any increase from playing as one(though I would take it with a grain of salt, most of the time I was a megafauna for only several minutes before round end or in very short bursts).

**Comment by Whoneedspacee** [comment]
Does quitting completely and reconnecting fix the lag?

**Comment by Taylorkills2** [comment]
> Does quitting completely and reconnecting fix the lag?

No, the only thing I've found that fixes it is when the server restarts for a new round.

**Comment by Dennok** [comment]
Ash drake lag curse.

**Comment by ExcessiveUseOfCobblestone** [comment]
I need something quantifiable that I can point to and say "this is getting worse after I do this" that isn't just "yeah i was lagging a bit after that mob died, and more after I killed the second one".

**Comment by projectkepler-ru** [comment]
still happens
